test/helper.rb

```diff
@@ -174,7 +174,7 @@ def version
     def with_acl
       admin = _new_client
       admin.acl('SETUSER', 'johndoe', 'on',
-                '+ping', '+select', '+command', '+cluster|slots', '+cluster|nodes',
+                '+ping', '+select', '+command', '+cluster|slots', '+cluster|nodes', '+readonly',
```

Comment:
This permission may be needed. I don't know why it becomes needed since when. Some cases became failure in the ACL test.